File: Tareas/Ejercicio12.rb
=begin Nombre: Sergio Alberto Hernandez Serrato Mat: 1853132 Hora: V5*
Programa que imprime la serie C
=end

letrero = ""
puts("Que cantidad de numeros va a generar la serie?(Solo enteros)")
numero = gets().to_i
numeropos = 0 
numeroneg= -1 
contador = 1
letrero = ""
i = 0
j = 0
k = 0
while i != numero
	while j != contador
		if i == numero
			break
		else
			numeropos = numeropos+ 2
			letrero = letrero+numeropos.to_s
			letrero = letrero+ ", "	
			i = i + 1 
			j = j + 1
		end
	end
	j = 0
	contador = contador+1	
	while k != 3
		if i == numero
			break
		else
			numeroneg = numeroneg-2
			letrero = letrero+ numeroneg.to_s
			letrero = letrero+ ", "
			i = i + 1
			k = k + 1
		end
	end
	k = 0
end
puts (letrero) 

File: Tareas/Ejercicio12.java
/*Nombre: Sergio Alberto Hernandez Serrato Mat: 1853132 Hora: V5*
Programa que imprime la serie C*/

import javax.swing.JOptionPane;
public class Ejercicio12{
	public static void main(String[] args) {
		int i,j,numero, numeropos = 0, numeroneg= -1, contador= 1;
		String letrero = "";
		String entrada = JOptionPane.showInputDialog("Ingresa el valor entero de la extension de la serie");
		numero = Integer. parseInt(entrada);
		for (i=0; i<numero;) 
		{
							for (j=0; j<contador; j++) 
							{
								if( i!= numero)
								{
									
									numeropos = numeropos+ 2;
									letrero = letrero+numeropos;
									letrero= letrero+ ", ";	
									i++;
								}
								else
								{
									break;
								}
								
							}
							contador = contador+1;
							for (j=0; j<3 ;j++ ) 
							{
								if(i != numero)
								{

									numeroneg = numeroneg-2;
									letrero = letrero+ numeroneg;
									letrero = letrero+ ", ";
									i++;
								}
								else
								{
									break;	
								}
									
							}
		}			
		JOptionPane.showMessageDialog(null, "La serie es: "+ letrero);	
	}
}